src/CommunityToolkit.Maui.Core/Views/Popup/MCTPopup.macios.cs

```diff
@@ -0,0 +1,176 @@
+﻿using CommunityToolkit.Maui.Core;
+using Microsoft.Maui.Handlers;
+using Microsoft.Maui.Platform;
+using UIKit;
+
+namespace CommunityToolkit.Core.Platform;
+
+/// <summary>
+/// The navite implementation of Popup control.
+/// </summary>
+public class MCTPopup : UIViewController
```

Comment:
sorry, I missed the previous comment. Can we extend https://github.com/CommunityToolkit/Maui/blob/main/src/CommunityToolkit.Maui.Core/Views/Alert/AlertView.macios.cs? I think it already allows to add content. we only need to add PopoverDelegate Dismissed.

Reply:
I don't think so. AlertView extends from `UIView`, here we extend from `UIViewController`

Reply:
let's skip it for now. we may back to it later